addons/voice/org.openhab.voice.marytts/README.md

```diff
@@ -25,3 +25,6 @@ marytts:bits1hsmm bits1-hsmm (de)
 
 The MaryTTS service produces audio streams using WAV containers and PCM (signed) codec with 16bit depth.
 The sample frequency depends on the chosen voice and ranges from 16kHz to 48kHz.
+
+## Log files
```

Comment:
There is a newline missing after this header

Reply:
Will fix that together with the thing in L30, once it is clarified :)